### ISSUE TITLE
-d flag doesn't seem to work

### ISSUE THREAD

**Issue by RunasSudo** [flamusdiu/python-pia#34]
As per the title, running e.g. `pia -a -d` does not seem to enable debug logging. There is no output from the command, whereas if `debug()` is inserted somewhere in `run.sh`, some debug output is produced.

**Comment by flamusdiu** [comment]
@RunasSudo Yeah, I never got to working that in properly. I'll need to actually fix it at some point. Sorry for the confusion.
